lib/features/onboarding/second_onboarding_screen.dart

```diff
@@ -0,0 +1,195 @@
+  late final PageController _pageController;
+  late final ValueNotifier<int> _currentPageNotifier;
+  static const int _numPages = 2;
+
+  @override
+  void initState() {
+    super.initState();
+    _currentPageNotifier = ValueNotifier<int>(0);
+    _pageController = PageController(initialPage: _currentPageNotifier.value);
+  }
+
+  @override
+  void dispose() {
+    _currentPageNotifier.dispose();
+    _pageController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          Flexible(
+            child: SizedBox(
+              child: ExpandablePageView(
+                controller: _pageController,
+                onPageChanged: (index) {
+                  _currentPageNotifier.value = index;
+                },
+                physics: const ClampingScrollPhysics(),
+                children: [
+                  _childPager(context, Assets.images.onboarding2,
+                      S.current.keepAllYourDiariesPrivate),
+                  _childPager(context, Assets.images.onboarding3,
+                      S.current.accessYourDiariesAnywhere),
+                ],
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(left: NartusDimens.padding32),
+            child: ValueListenableBuilder<int>(
+              valueListenable: _currentPageNotifier,
+              builder: (context, pageIndex, _) {
+                return DotsIndicator(
+                    dotsCount: _numPages,
+                    position: pageIndex.toDouble(),
+                    decorator: const DotsDecorator(
+                      activeColor: NartusColor.primary,
+                      color: Colors.grey,
+                      activeSize: Size.square(8.0),
+                      size: Size.square(8.0),
+                    ));
+              },
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(
+                NartusDimens.padding32,
+                NartusDimens.padding30,
+                NartusDimens.padding32,
+                NartusDimens.padding40),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
+                SizedBox(
+                  child: NartusButton.primary(
+                    label: S.current.continueWithEmail,
+                    iconPath: Assets.images.idSmsIcon,
+                    iconSemanticLabel: '',
+                    onPressed: () {},
+                  ),
+                ),
+                const SizedBox(height: NartusDimens.padding12),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    Expanded(
```

Comment:
You can use Padding instead of Expanded and then SizedBox

Reply:
leave it here, NartusButton.secondary is updated, we can improve this code